hasher-matcher-actioner/hmalib/fetching/fetcher.py

```diff
@@ -91,6 +91,13 @@ def _get_all_active_collabs(
             if c.to_pytx_collab_config().enabled
         ]
 
+    def get_store(
```

Comment:
This style allows overriding when we are setting up tests.

Reply:
(ignorable): This would make a good docstring